downstream/modules/platform/con-aap-migration-prereqs.adoc

```diff
@@ -0,0 +1,16 @@
+[id="aap-migration-prereqs"]
+
+= Prerequisites
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
+
+[role="_abstract"]
+
+* Secret key secret
+* Postgresql configuration
+* Role-Based Access Controls
+* Network connectivity
+
+[NOTE]
+====
+Secret key information can be located in the inventory file created during the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link::https://access.redhat.com/[Ansible support] via the Red Hat Customer portal. 
```

Comment:
Looks like the Ansible support link is broken, or maybe that is just because this is a preview rendering.

Reply:
Suggested changes complete.